src/altinn-app-frontend/src/features/form/containers/Form.tsx

```diff
@@ -98,34 +98,41 @@ export function Form() {
         }
         renderedInGroup = renderedInGroup.concat(childList);
       });
+      const componentsToRender = layout.filter((component) => (!renderedInGroup.includes(component.id) && component.type !== 'NavigationBar'));
+      setFilteredLayout(componentsToRender);
+      setNavComponent(layout.find((c) => c.type === 'NavigationBar') as ILayoutComponent);
     }
-
-    if (layout) {
-      componentsToRender = layout.filter((component) => {
-        return !renderedInGroup.includes(component.id);
-      });
-    }
-    setFilteredLayout(componentsToRender);
   }, [layout]);
 
   return (
     <div>
-      {hasRequiredFields(layout) &&
-        <MessageBanner
-          language={language}
-          error={requiredFieldsMissing}
-          messageKey={'form_filler.required_description'}
-        />
+      {navComponent &&
```

Comment:
Maybe this can be a landmark as well?
You can use the same condition to consider adding it to the landmarkShortcuts-array.

Reply:
Don't think it is needed. The point of having a "skip to main content" shortcut is to jump over repeating stuff like this.

Reply:
I believe you are right. It's probably more common on webpages with more boilerplate and advertisement before the page nav etc.